controllers/front/CartController.php

```diff
@@ -621,8 +657,12 @@ protected function areProductsAvailable()
 
             if ($currentProduct->hasAttributes() && $product['id_product_attribute'] === '0') {
                 return $this->trans(
-                   'The item %product% in your cart is now a product with attributes. Please delete it and choose one of its combinations to proceed with your order.',
-                    ['%product%' => $product['name']],
+                    'The product [1]%product%[/1] is now a product with combinations. Please select one to proceed with your order.',
```

Comment:
Can attributes be selected at this point in the cart? If not then the second part should stay as before: `Please delete it and choose one of its combinations to proceed with your order.`